File: php/Product.php
<?php

require_once 'Shirt.php';

class Product extends Shirt {
    private $idProduct;
    private $name;
    private $brand;
    private $price;

    // Parameterized constructor
    public function __construct($size = 0, $material = "", $gender = "", $color = "", $sleeveType = "", $idProduct = "", $name = "", $brand = "", $price = 0) {
        parent::__construct($size, $material, $gender, $color, $sleeveType);
        $this->idProduct = $idProduct;
        $this->name = $name;
        $this->brand = $brand;
        $this->price = $price;
    }

    // Getters
    public function getIdProduct() {
        return $this->idProduct;
    }

    public function getName() {
        return $this->name;
    }

    public function getBrand() {
        return $this->brand;
    }

    public function getPrice() {
        return $this->price;
    }

    // Setters
    public function setIdProduct($idProduct) {
        $this->idProduct = $idProduct;
    }

    public function setName($name) {
        $this->name = $name;
    }

    public function setBrand($brand) {
        $this->brand = $brand;
    }

    public function setPrice($price) {
        $this->price = $price;
    }
}

?>

File: python/Product.py
from Shirt import Shirt

class Product(Shirt):
    def __init__(self, size="", material="", gender="", color="", sleeve_type="", product_id=0, name="", brand="", price=0):
        super().__init__(size, material, gender, color, sleeve_type)
        self.product_id = product_id
        self.name = name
        self.brand = brand
        self.price = price

    def get_product_id(self):
        return self.product_id

    def set_product_id(self, product_id):
        self.product_id = product_id

    def get_name(self):
        return self.name

    def set_name(self, name):
        self.name = name

    def get_brand(self):
        return self.brand

    def set_brand(self, brand):
        self.brand = brand

    def get_price(self):
        return self.price

    def set_price(self, price):
        self.price = price
